internal/command/status/machines.go

```diff
@@ -189,11 +189,16 @@ func renderPGStatus(ctx context.Context, app *api.AppCompact, machines []*api.Ma
 		client   = client.FromContext(ctx).API()
 	)
 
-	if postgres.IsFlex(machines[0]) {
-		yes, note := isQuorumMet(machines)
-		if !yes {
-			fmt.Fprintf(io.Out, colorize.Yellow(note))
+	if len(machines) > 0 {
+		if postgres.IsFlex(machines[0]) {
+			yes, note := isQuorumMet(machines)
+			if !yes {
+				fmt.Fprintf(io.Out, colorize.Yellow(note))
+			}
 		}
+	} else {
+		fmt.Fprintf(io.Out, "No machines are available on this app %s\n", app.Name)
+		return
```

Comment:
no need to change now but this is a common anti pattern in go code, it is much easier to follow the flow as:

```
if len(machines) == 0 {
  return
} 

.. continue the normal flow here.
```

Reply:
I had no idea, though when you say it it makes since. I'll keep that in mind, thank you